<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Пользователь</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0,0,0,5">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(118, 227, 131);</string>
               </property>
               <property name="text">
                <string>Персональныe данные</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="lbl_role">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 227, 131);</string>
                 </property>
                 <property name="text">
                  <string>Инфо</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../icons/person.jpg</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Заказы</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>Comic Sans MS</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(118, 227, 131);</string>
               </property>
               <property name="text">
                <string>Создание заказа</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 227, 131);</string>
                 </property>
                 <property name="text">
                  <string>Введите номера услуг (через запятую):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_service"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 227, 131);</string>
                 </property>
                 <property name="text">
                  <string>Код клиента</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="edit_client">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <number>999999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 227, 131);</string>
                 </property>
                 <property name="text">
                  <string>Код заказа:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_order"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 227, 131);</string>
                 </property>
                 <property name="text">
                  <string>Введите время проката</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_time"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_add_order">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(73, 140,81);</string>
             </property>
             <property name="text">
              <string>Сформировать заказ</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_clients">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(73, 140,81);</string>
             </property>
             <property name="text">
              <string>Клиенты</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_serv">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(73, 140,81);</string>
             </property>
             <property name="text">
              <string>Услуги</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_ord">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(73, 140,81);</string>
             </property>
             <property name="text">
              <string>Заказы</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTabWidget" name="tabWidget_3">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_7">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
          </property>
          <attribute name="title">
           <string>Пользователь</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14" stretch="0,0,0,0,5">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(118, 227, 131);</string>
               </property>
               <property name="text">
                <string>Персональные данные</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="lbl_role2">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 227, 131);</string>
                 </property>
                 <property name="text">
                  <string>Инфо</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../icons/person.jpg</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Отчёты</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(118, 227, 131);</string>
             </property>
             <property name="text">
              <string>Отчёт</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(118, 227, 131);</string>
               </property>
               <property name="text">
                <string>Период времени с</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="start_date">
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2022</year>
                 <month>3</month>
                 <day>1</day>
                </datetime>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(118, 227, 131);</string>
               </property>
               <property name="text">
                <string>по</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="end_date">
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2022</year>
                 <month>4</month>
                 <day>30</day>
                </datetime>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_service">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(73, 140,81);</string>
             </property>
             <property name="text">
              <string>Кол-во услуг по дням</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_order">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(73, 140,81);</string>
             </property>
             <property name="text">
              <string>Кол-во заказов по дням</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>История входа</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(118, 227, 131);</string>
             </property>
             <property name="text">
              <string>История входа</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_exit">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(73, 140,81);</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
